network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/CalculatedBus.java

```diff
@@ -125,16 +122,118 @@ public VoltageLevel getVoltageLevel() {
 
     @Override
     public double getV() {
-        return getAttributes().getV();
```

Comment:
You have to synchronize bus angle value too ... in getAngle and setAngle methods

Reply:
I added synchronization for Angle